crowdfunding/crowdfunding/settings.py

```diff
@@ -20,27 +20,44 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'django-insecure-6f5fiv53l$d=%d_0_8&znvd!6&d3rfy-qowzswx^u)i-p_dsm6'
-
-# SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+SECRET_KEY = os.environ.get(
+    'DJANGO_SECRET_KEY',
+    'django-insecure-6f5fiv53l$d=%d_0_8&znvd!6&d3rfy-qowzswx^u)i-p_dsm6'
+)
 
-ALLOWED_HOSTS = []
 
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = os.environ.get("DJANGO_DEBUG", "False" != "False")
+DEBUG = False
+# ALLOWED_HOSTS = ['floral-wave-2596.fly.dev']
+ALLOWED_HOSTS = ['*']
```

Comment:
this is not really best practice, it's quite insecure, you wouldn't want to do this if you were planning to make this application for public use.